docs/architecture/mobile-clients/ios/push-notification-troubleshooting.md

```diff
@@ -0,0 +1,98 @@
+---
+sidebar_custom_props:
+  access: bitwarden
+sidebar_position: 2
+---
+
+# iOS Push Notification Troubleshooting Tips
+
+## Overview
+
+We use push notifications in the Password Manager app for two purposes:
```

Comment:
1. Thought it would be a good addition to link where server defines [push notification types](https://github.com/bitwarden/server/blob/main/src/Core/Enums/PushType.cs) and while doing so, it made me remember we use push for logging out all devices too. 
2. 🤔 It's common for someone new to assume changes like client settings are synced, not sure if we need to be more specific here about vault changes meaning cipher / folder / org CRUD operations.

Reply:
1. Oh! Good call!
2. We probably should; that was a subtlety I hadn't anticipated